### ISSUE TITLE
support Anaconda on MacOS?

### ISSUE THREAD

**Issue by stevengj** [willwhitney/hydrogen#118]
I have Jupyter (and ZeroMQ) installed via Anaconda on MacOS.  Is there a way to use that rather than pip and homebrew?

**Comment by willwhitney** [comment]
ZeroMQ: I'm not sure. If you install `pkg-config` and make sure Anaconda's ZeroMQ install is in your `$PATH`, I would sort of expect it to work. But honestly I don't have any special insight about how the [Node `zmq` library](https://github.com/JustinTulloss/zeromq.node) goes about finding your ZeroMQ install.

Jupyter: If you run (pick based on how you normally launch Jupyter) `<jupyter | ipython> kernelspec install-self`, that'll create a folder in `/usr/local/share/jupyter/kernels` named "python" or "python3". This folder contains a file called kernel.json, which tells Hydrogen or any other frontend how to launch the Jupyter kernel for Python. If it doesn't point at the correct Python install, you can edit the command in the `argv` field in there to be the right command for launching your Python kernel.

**Comment by stevengj** [comment]
Note that `pkg-config` looks in `PKG_CONFIG_PATH`, not `PATH`.  (I already had the kernelspecs installed, so that's not an issue for me.)

Anaconda doesn't seem to include `pkg-config`.  So, I used the Homebrew `pkg-config`, which is in `/usr/local/bin`.  But then I ran into atom-community/linter#150: running Atom via the GUI didn't have /usr/local/bin in my `PATH`, so I had to add `process.env.PATH = ["/usr/local/bin", process.env.PATH].join(":")` to my `init.coffee` file.

But it sounds like this is a node zmq and Atom problem, not your problem.
